newrelic/config.go

```diff
@@ -8,10 +8,11 @@ import (
 	"log"
 	"net/url"
 
-	synthetics "github.com/dollarshaveclub/new-relic-synthetics-go"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/logging"
 	"github.com/hashicorp/terraform-plugin-sdk/helper/pathorcontents"
 	insights "github.com/newrelic/go-insights/client"
+	nrConfig "github.com/newrelic/newrelic-client-go/pkg/config"
+	synthetics "github.com/newrelic/newrelic-client-go/pkg/synthetics"
```

Comment:
you shouldn't have to alias the package (since it's already called `synthetics`)